### PR TITLE
Add Dynatrace Ruxit framework

### PR DESCRIPTION
**Pull request by aloismayr**
Adds Dynatrace Ruxit Agent Framework

### CODE REVIEW

**Comment by jgawor** [comment]
Thank you so much for your PR! Can you please follow the process to sign the individual or corporate CLA and resolve the test failure?

**Comment by aloismayr** [comment]
@jgawor We (Dynatrace) already has a CLA in place with my account listed as contributor to cloudfoundry organization.

**Comment by jgawor** [comment]
@aloismayr looks like @cfdreddbot still doesn't recognize your id. @chipchilders maybe you can help?

**Comment by aloismayr** [comment]
@jgawor I can send the CLA again. Are you at cloudfoundry.slack.com ?

**Comment by chipchilders** [comment]
@aloismayr please resend the cla to contributors@cloudfoundry.org. I've checked the system and our filing location for any reference to Dynatrace. It's not there. It's possible that Dynatrace provided a CLA to Pivotal, but that is different from a CFF CLA.

Sorry for the issue. If you can forward again, I'll ensure that it's processed quickly and that this PR is updated.

**Comment by chipchilders** [comment]
open / close to check CLA

**Comment by chipchilders** [comment]
open / close to check CLA

**Comment by chipchilders** [comment]
@jgawor we have the CLA that covers @aloismayr ... pull away!

**Comment by jgawor** [comment]
Awesome - thank you @chipchilders!

File: docs/framework-ruxit_agent.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The framework can be configured by modifying the [`config/ruxit_agent.yml`][] file in the buildpack fork. The framework uses the [`Repository` utility support][repositories] and so it supports the [version syntax][] defined there.
+
+| Name | Description
+| ---- | -----------
+| `repository_root` | The URL of the Ruxit repository index ([details][repositories]).
+| `version` | The version of Ruxit to use. Candidate versions can be found in [this listing][].
+
+## Support
+This buildpack is currently Beta. If you have any questions or problems regarding the build pack itself please don't hesitate to contact Dynatrace Ruxit on https://answers.ruxit.com/, be sure to use "cloudfoundry" as a topic.
```
Comment (jgawor) [inline docs/framework-ruxit_agent.md]: The buildpack is not in beta - maybe you mean the Ruxit integration is? Can you update this statement?

**Comment by jgawor** [comment]
Everything looks great but there is one confusing statement under the "support" section in the doc. Can you clarify that?

**Comment by aloismayr** [comment]
@jgawor Sorry for the confusion. The integration is Beta right now, the buildpack of course is not.

**Comment by jgawor** [comment]
Merged. Thank you so much for your contribution!

**Pull request merged**